neo/core/group.py

```diff
@@ -0,0 +1,58 @@
+"""
+This module implements :class:`Group`, which represents a subset of the
+channels in an :class:`AnalogSignal` or :class:`IrregularlySampledSignal`.
+
+It replaces and extends the grouping function of the former :class:`ChannelIndex`
+and :class:`Unit`.
+"""
+
+from neo.core.container import Container
+
+
+
+class Group(Container):
+    """
+    Can contain any of the data objects, views, or other groups,
+    outside the hierarchy of the segment and block containers.
+    A common use is to link the :class:`SpikeTrain` objects within a :class:`Block`,
+    possibly across multiple Segments, that were emitted by the same neuron.
+
+    *Required attributes/properties*:
+        None
+
+    *Recommended attributes/properties*:
+        :objects: (Neo object) Objects with which to pre-populate the :class:`Group`
```

Comment:
need to specify of which type: only AnalogSignal and SpikeTrain, right?

Reply:
no, a Group can contain any Neo object.